packages/test/test-end-to-end-tests/src/test/blobsisAttached.spec.ts

```diff
@@ -143,8 +145,9 @@ describeNoCompat("blob handle isAttached", (getTestObjectProvider) => {
 			await forceWriteMode(map, dataStore1);
 
 			const blob = await dataStore1.runtime.uploadBlob(stringToBuffer(testString, "utf-8"));
-			const pendingBlobs = (runtimeOf(dataStore1).getPendingLocalState() as PendingLocalState)
-				.pendingAttachmentBlobs;
+			const pendingBlobs = (
```

Comment:
you can make this a bit cleaner
```suggestion
			const pendingBlobs = await runtimeOf(dataStore1).getPendingLocalState()
			       .then((s)=> (s as PendingLocalState).pendingAttachmentBlobs);
```

Reply:
not totally sure typing the s param like that will work, it not, do it in the lamda body:
```
(s)=> (s as PendingLocalState).pendingAttachmentBlobs)
```